network-api/networkapi/wagtailpages/pagemodels/customblocks/listing.py

```diff
@@ -10,8 +10,10 @@ class ListingCard(blocks.StructBlock):
     alt_text = blocks.CharBlock(required=False, help_text="Alt text for card's image.")
 
     title = blocks.CharBlock(help_text="Heading for the card.")
+    category = blocks.CharBlock(help_text="Category text for the card.", required=False)
+    date_meta = blocks.CharBlock(help_text="Date and time or other information.", required=False)
```

Comment:
Do the label "category" and "Date meta" make sense in other contexts where this block is used (if it is)? Can/should we maybe use terminology that is a bit more decoupled from the content? I think "metadata" is good. Maybe "highlighted metadata" for the blue part?